#! /bin/sh /usr/share/dpatch/dpatch-run
## Backport OutOfMemoryError exception
##
## All lines beginning with `## DP:' are a description of the patch.

@DPATCH@
diff -aur wireshark-0.99.4.orig/epan/exceptions.h wireshark-0.99.4/epan/exceptions.h
--- wireshark-0.99.4.orig/epan/exceptions.h	2006-10-31 18:59:15.000000000 +0100
+++ wireshark-0.99.4/epan/exceptions.h	2008-11-29 13:43:44.000000000 +0100
@@ -68,6 +68,13 @@
 #define ScsiBoundsError		5	
 
 
+/**
+    Running out of memory.
+    A dissector tried to allocate memory but that failed.
+**/
+#define OutOfMemoryError        6
+
+
 /* Usage:
  *
  * TRY {
Only in wireshark-0.99.4/epan: exceptions.h~
